google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/ConnectionWorkerPoolTest.java

```diff
@@ -315,6 +321,68 @@ public void testMultiStreamAppend_appendWhileClosing() throws Exception {
     assertThat(connectionWorkerPool.getTotalConnectionCount()).isEqualTo(0);
   }
 
+  @Test
+  public void testBlocking() throws Exception {
```

Comment:
Is this test hitting the issue and how? Clean up the test?

Reply:
Done